login.js

```diff
@@ -0,0 +1,81 @@
+const emailInput = document.getElementById("userName");
+const passwordInput = document.getElementById("userPassword");
+
+const errorMessage = document.getElementById("error-message");
+const errorPassword = document.getElementById("error-password");
+
+const loginBtn = document.getElementById("loginBtn");
+
+document.addEventListener("DOMContentLoaded", () => {
+  emailInput.addEventListener("input", () => {
+    if (emailInput.value !== "" && emailInput.value.includes("@")) {
+      emailInput.classList.remove("error");
+      errorMessage.textContent = "";
+      errorMessage.style.display = "none";
+    }
+    updateButtonState();
+  });
+
+  passwordInput.addEventListener("input", () => {
+    if (passwordInput.value.length >= 8) {
+      passwordInput.classList.remove("error");
+      errorPassword.textContent = "";
+      errorPassword.style.display = "none";
+    }
+    updateButtonState();
+  });
+
+  emailInput.addEventListener("focusout", () => {
+    if (emailInput.value === "") {
+      emailInput.classList.add("error");
+      errorMessage.textContent = "잘못된 이메일입니다.";
+      errorMessage.style.display = "block";
+    } else if (!emailInput.value.includes("@")) {
+      emailInput.classList.add("error");
+      errorMessage.textContent = "잘못된 이메일 형식입니다.";
+      errorMessage.style.display = "block";
+    } else {
+      emailInput.classList.remove("error");
+      errorMessage.textContent = "";
+      errorMessage.style.display = "none";
+    }
+    updateButtonState();
+  });
+
+  passwordInput.addEventListener("focusout", () => {
+    if (passwordInput.value === "") {
+      passwordInput.classList.add("error");
+      errorPassword.textContent = "비밀번호를 입력해주세요";
+      errorPassword.style.display = "block";
+    } else if (passwordInput.value.length < 8) {
+      passwordInput.classList.add("error");
+      errorPassword.textContent = "비밀번호를 8자 이상 입력해주세요.";
+      errorPassword.style.display = "block";
+    } else {
+      passwordInput.classList.remove("error");
+      errorPassword.textContent = "";
+      errorPassword.style.display = "none";
+    }
+    updateButtonState();
+  });
+
+  loginBtn.addEventListener("click", function (e) {
+    e.preventDefault();
+    if (!loginBtn.disabled) {
+      window.location.href = "/item";
+    }
+  });
```

Comment:
💊 제안  
form이 유효하지 않으면 submit button이 disabled 상태라 제출이 되지 않으므로 관련 코드는 없어도 될 것 같아요~
```suggestion
  loginBtn.addEventListener("click", function (e) {
    e.preventDefault();
    window.location.href = "/item";
  });
```